### PR TITLE
[CMTOOL-353] added contributing, CoC, security file and codeowners, f…

### PR DESCRIPTION
**Pull request by istudens**
…ixed readme

Issue: https://issues.redhat.com/browse/CMTOOL-353

### CODE REVIEW

**Comment by istudens** [comment]
@emmartins @fl4via @fjuma can you review this please? thanks

**Review by emmartins — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Security Policy
```
Comment (fjuma) [inline SECURITY.md:1]: This PR looks great to me! Just have one small comment about this file. WildFly recently tweaked it's SECURITY.md file so I think we can copy the new content from there:

https://github.com/wildfly/wildfly/blob/main/SECURITY.md

Comment (istudens) [reply SECURITY.md]: SECURITY.md has been updated according the the current content of WildFly.

**Review by fjuma — APPROVED** [review]
Thanks @istudens!

**Comment by emmartins** [comment]
TY @istudens

**Pull request merged**